sycl/include/CL/sycl.hpp

```diff
@@ -11,6 +11,8 @@
 #include <CL/sycl/accessor.hpp>
 #include <CL/sycl/atomic.hpp>
 #include <CL/sycl/backend.hpp>
+#include <CL/sycl/backend/cuda.hpp>
```

Comment:
Why are you adding these?
Backends are optional in SYCL-2020 (https://github.com/KhronosGroup/SYCL-Shared/blob/master/proposals/sycl_generalization.md):

>  The header SYCL/sycl.hpp will provide all the generic SYCL Next API. SYCL users should not include any specific backend headers if they use only generic SYCL API. Backend specific API (e.g. interoperability API) is available in SYCL/backend/backend_name.hpp, e.g. OpenCL backend headers will be in SYCL/backend/opencl.hpp. Vendors can provide their own backends. The interoperability API is defined in SYCL/vendor/backend/backend_name/backend_header.hpp SYCL vendor backends are namespaced as sycl::vendor::backend::vendor_name::backend_name.

Reply:
Oh, OK, then.
Removed these.